plugin.json

```diff
@@ -28,7 +28,39 @@
         "display_name": "Webhook Secret",
         "type": "generated",
         "help_text": "The secret used to authenticate the webhook to Mattermost.",
-        "regenerate_help_text": "Regenerates the secret for the webhook URL endpoint. Regenerating the secret invalidates your existing JIRA integrations."
+        "regenerate_help_text": "Regenerates the secret for the webhook URL endpoint. Regenerating the secret invalidates your existing Jira integrations."
+      },
+      {
+        "key": "RolesAllowedToEditJiraSubscriptions",
+        "display_name": "Roles Allowed to Edit Jira Subscriptions",
```

Comment:
```suggestion
        "display_name": "Mattermost Roles Allowed to Edit Jira Subscriptions",
```

Reply:
> Propose minor updates to make it clear which setting is for Mattermost roles/groups, which for Jira.
> 
> @aaronrothschild doesn't block merging this PR, but we should consider revisiting before we ship to ensure it's clear what happens if only one or both settings are configured. For instance, if I choose MM channel admins and list two Jira groups, will the allowed users be an intersection or a union of the two groups of users? This could just be a note, help text update or a doc.

I think a note to the effect of "Users must be member of at least one of the jira groups and have appropriate mattermost permissions in order to setup a subscription" would be effective enough.